handbook/sales/onboarding/joining_customer_calls.md

```diff
@@ -0,0 +1,25 @@
+# Joining customer calls
```

Comment:
It's annoying to go make this change everywhere, but could we at least change them on this page to "customer and prospect" calls? Important distinction to use internally.

Reply:
done (everywhere on this page)!